assets/styles/layout/_alerts.scss

```diff
@@ -8,17 +8,22 @@
     margin: 0;
   }
 
-  a{
+  a {
     font-weight: bold;
   }
 
+  i {
+    font-size: 0.8rem;
+    vertical-align: middle;
```

Comment:
bleh, just noticed this doesn't work in firefox. appears to work fine in chromium browsers. not a big deal, probably, but might want to try to adjust it later